### ISSUE TITLE
api-platform/core version conflict disallows upgrade past CVE version

### ISSUE THREAD

**Issue by stefandoorn** [Sylius/Sylius#14863]
**Sylius version affected**: 1.11 + 1.12 and probably lower

**Description**  
`api-platform/core` v2.6 has a security advisory out: https://github.com/api-platform/core/security/advisories/GHSA-vr2x-7687-h6qv

Sylius v1.11 + v1.12 requires v2.6 and conflicts with v2.7 - there's several issues about the conflict and breaking changes. 

But, currently everyone e.g. on Sylius 1.11 / 1.12 and that does run security scans (e.g. with `symfony security:check`) runs into an error as there is a CVE out for this unsafe version.

Basically you are then stuck, as Sylius 1.13 is not yet out. 

In v1.10 this conflict is not there, but probably due to some breaking change many users will have that version conflicted in their local versions.

**Comment by jakubtobiasz** [comment]
Hi @stefandoorn 🙋🏼‍♂️!
Notice the conflict is only for version `2.7.0`. You can use the latest 2.7.x version on your project :).
![CleanShot 2023-03-21 at 10 29 10](https://user-images.githubusercontent.com/80641364/226565592-7100a55d-d29e-4e74-83b2-4e6cc1274f5b.png)

**Comment by stefandoorn** [comment]
Ahhh overlooked that, sorry!
